plugins/org.komodo.eclipse.workspace/src/org/komodo/eclipse/workspace/workspace-core-plugin-repository-in-memory-config.json

```diff
@@ -0,0 +1,84 @@
+{
+    "name" : "Komodo Local Repository",
```

Comment:
Why is this being committed, especially as it is not the 'in-memory' version since its got a cache